### PR TITLE
[Event Hubs] Doc Comment Updates

### PR DESCRIPTION
**Pull request by jsquire**
# Summary

The focus of these changes is to apply some learnings from the recent UX studies for event publishing to improve documentation that was unclear to participants.

### CODE REVIEW

File: sdk/eventhub/Azure.Messaging.EventHubs/src/EventData.cs [hunk 1]
```diff
@@ -59,8 +58,11 @@ public class EventData
         /// </value>
         ///
         /// <remarks>
-        ///   The <see cref="ContentType" /> is intended to allow coordination between event
-        ///   producers and consumers.  It has no meaning to the Event Hubs service.
+        ///   The <see cref="ContentType" /> is managed by the application and is intended to
+        ///   allow coordination between event producers and consumers.
+        ///
```
Comment (christothes) [inline sdk/eventhub/Azure.Messaging.EventHubs/src/EventData.cs:63]: Does this spacing get rendered without the `<para>` element?

File: sdk/eventhub/Azure.Messaging.EventHubs/src/EventData.cs [hunk 2]
```diff
@@ -166,13 +177,14 @@ public string CorrelationId
...
-        ///   during Event Hubs operations.
+        ///   The set of free-form properties which may be used for associating metadata with the event that
+        ///   is meaningful within the application context.
         /// </summary>
         ///
         /// <remarks>
-        ///   A common use case for <see cref="EventData.Properties" /> is to associate serialization hints for the <see cref="EventData.EventBody" />
-        ///   as an aid to consumers who wish to deserialize the binary data.
+        ///   A common use case for <see cref="EventData.Properties" /> is to associate serialization hints
+        ///   for the <see cref="EventData.EventBody" /> as an aid to consumers who wish to deserialize the binary data.
```
Comment (christothes) [inline sdk/eventhub/Azure.Messaging.EventHubs/src/EventData.cs:186]:
```suggestion
        ///   for the <see cref="EventData.EventBody" /> as an aid to consumers who wish to deserialize the binary data
```

**Review by christothes — APPROVED** [review]

File: sdk/eventhub/Azure.Messaging.EventHubs/src/EventData.cs (hunk 1 above)
Comment (jsquire) [reply sdk/eventhub/Azure.Messaging.EventHubs/src/EventData.cs:63]: Yup;   I fixed the doc tools for that one, personally.  😄

Comment (jsquire) [reply sdk/eventhub/Azure.Messaging.EventHubs/src/EventData.cs:63]: https://github.com/docascode/ECMA2Yaml/pull/190

**Pull request merged**